libraries/AP_Scripting/docs/docs.lua

```diff
@@ -2354,6 +2354,25 @@ function vehicle:set_circle_rate(rate_dps) end
 ---@return number|nil
 function vehicle:get_circle_radius() end
 
+-- add an offset to vehicle's target position, velocity and acceleration in autonomous modes
+---@param pos_offset_NED Vector3f_ud
+---@param vel_offset_NED Vector3f_ud
+---@param accel_offset_NED Vector3f_ud
+---@return boolean
+function vehicle:set_posvelaccel_offset(pos_offset_NED, vel_offset_NED, accel_offset_NED) end
```

Comment:
If you add a `AC_PosControl` singleton you can go direct. You would have to depends copter on the bindings. That means 0 cost on other vehicles and we can enable it on plane easily if we wanted to.

Reply:
done, thanks